### PR TITLE
Update Import-CsOnlineAudioFile.md

### PR DESCRIPTION
**Pull request by carolynblanding**
@kenwith cc: @roykuntz 

Updating to example 1 to refer to related AA cmdlet where stored variable is used. Added example 2 for call queue with reference to the set-cscallqueue cmdlet which was also updated to correctly reflect need to provide ID using stored variable.

### CODE REVIEW

File: skype/skype-ps/skype/Import-CsOnlineAudioFile.md [hunk 1]
```diff
@@ -32,7 +32,16 @@ $content = Get-Content "C:\Media\Welcome.wav" -Encoding byte -ReadCount 0
 $audioFile = Import-CsOnlineAudioFile -ApplicationId "OrgAutoAttendant" -FileName "Hello.wav" -Content $content
 ```
 
-This example creates a new audio file using the WAV content that has a filename of Hello.wav to be used with organizational auto attendants.
+This example creates a new audio file using the WAV content that has a filename of Hello.wav to be used with organizational auto attendants. The stored variable, $audioFile, will be used when running other cmdlets to update the audio file for Auto Attendant, for example: [New-CsAutoAttendantPrompt] (https://docs.microsoft.com/en-us/powershell/module/skype/new-csautoattendantprompt?view=skype-ps)
```
Comment (tseward) [inline skype/skype-ps/skype/Import-CsOnlineAudioFile.md:35]: `en-us` needs to be removed.

File: skype/skype-ps/skype/Import-CsOnlineAudioFile.md [hunk 2]
```diff
@@ -32,7 +32,16 @@ $content = Get-Content "C:\Media\Welcome.wav" -Encoding byte -ReadCount 0
...
+
+### -------------------------- Example 2 --------------------------
+```powershell
+$content = Get-Content "C:\Media\MOH.wav" -Encoding byte -ReadCount 0
+$audioFile = Import-CsOnlineAudioFile -ApplicationId "HuntGroup" -FileName "MOH.wav" -Content $content
+```
+
+This example creates a new audio file using the WAV content that has a filename of MOH.wav to be used as a Music On Hold file with a Call Queue. The stored variable, $audioFile, will be used with [Set-CsCallQueue] (https://docs.microsoft.com/en-us/powershell/module/skype/set-cscallqueue?view=skype-ps) to provide the audio file id.
```
Comment (tseward) [inline skype/skype-ps/skype/Import-CsOnlineAudioFile.md:43]: Also remove `en-us` from here.

File: skype/skype-ps/skype/Import-CsOnlineAudioFile.md (hunk 1 above)
Comment (dariomws) [inline skype/skype-ps/skype/Import-CsOnlineAudioFile.md:35]:
```suggestion
This example creates a new audio file using the WAV content that has a filename of Hello.wav to be used with organizational auto attendants. The stored variable, $audioFile, will be used when running other cmdlets to update the audio file for Auto Attendant, for example: [New-CsAutoAttendantPrompt] (https://docs.microsoft.com/powershell/module/skype/new-csautoattendantprompt)
```

File: skype/skype-ps/skype/Import-CsOnlineAudioFile.md (hunk 2 above)
Comment (dariomws) [inline skype/skype-ps/skype/Import-CsOnlineAudioFile.md:43]:
```suggestion
This example creates a new audio file using the WAV content that has a filename of MOH.wav to be used as a Music On Hold file with a Call Queue. The stored variable, $audioFile, will be used with [Set-CsCallQueue] (https://docs.microsoft.com/powershell/module/skype/set-cscallqueue) to provide the audio file id. 
```

**Pull request merged**